src/js/components/CombinationMenuItem.vue

```diff
@@ -1,16 +1,16 @@
 <template>
   <div>
-    <span class="clickable-menu-item" @click="open = !open">{{name}}{{user ? ` - ${username} - v. ${version}` : ''}}</span>
+    <span class="clickable-menu-item" @click="selectCombination">{{name}}{{user ? ` - ${username} - v. ${version}` : ''}}</span>
     <i class="fa fa-clone" @click="cloneScroll"></i>
     <i v-show="locked" class="fa fa-lock" style="color: red"></i>
     <i v-show="!locked" class="fa fa-unlock" style="color: green"></i>
     <div class="children" v-show="open">
         <ul>
-          <li v-if="menuType === 'text'" v-for="child in children">
+          <li v-if="menuType === 'text'" v-for="child in children[menuType]">
             <column-menu-item :data-id="child.id"
                             :name="child.name"></column-menu-item>
```

Comment:
Alignment looks wonky for the `column-menu-item`.

Something like: 
```
<column-menu-item
  :data-id="child.id"
  :name="child.name">
</column-menu-item>
```

Reply:
Agreed, I'll give it a clean up.